src/components/inputBoxContactUs.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+const InputBoxName = () => {
+  const inputBoxName = 'Your name';
+  const inputBoxNameStyle = {
+    fontSize: '20px'
+  };
+
+  return <input
+    placeholder={ inputBoxName }
+    style={ inputBoxNameStyle } />
+}
+
+const InputBoxEmail = () => {
+  const inputBoxEmail = 'Email placeholder';
+  const inputBoxEmailStyle = {
+    fontSize: '20px'
+  };
+
+  return <input
+    placeholder={ inputBoxEmail }
+    style={ inputBoxEmailStyle } />
+}
+
+
+const InputBoxMessage = () => {
+  const inputBoxMessage = 'Write your message';
+  const inputBoxMessageStyle = {
+    fontSize: '100px'
+  };
+
+  return <input
+    placeholder={ inputBoxMessage }
+    style={ inputBoxMessageStyle } />
+
+};
+
+
+export default InputBoxName;
+export default InputBoxEmail;
+export default InputBoxMessage;
```

Comment:
![image](https://user-images.githubusercontent.com/24725508/85300743-3066e380-b464-11ea-8571-7036b9525ed7.png)

Hello 🙂  I tried to pull down your branch and this error came up. I think if we change the 3 default exports to be module exports (e.g. `export const InputBoxMessage`) then I think it should work!